<launch>
    <arg name="bot" default="false" />
    <group unless="$(arg bot)"> <!-- when bot is false: invoke simulation -->       
        <include file="$(find simple_sim_2mats)/launch/2mats.launch" />
		<!-- My Code Added -->
		<node name="stop_at_edge" pkg="two_mats_pkg" type="stop_at_edge.py" required="true" output="screen">
			<param name="img_topic_name"   type="string" value="camera_view" />   
			<param name="twist_topic_name" type="string" value="cmd_vel" />  
		</node>
    </group>

    <group if="$(arg bot)"> <!-- when bot is true: invoke Prizm bot -->
        <include file="$(find prizm_ros)/launch/twist_controller.launch" />
		<!-- My Code Added -->
		<node name="stop_at_edge" pkg="two_mats_pkg" type="stop_at_edge.py" required="true" output="screen">
			<param name="img_topic_name"   type="string" value="cam_pub/image_raw" />   
			<param name="twist_topic_name" type="string" value="prizm/twist_controller/twist_cmd" />  
		</node>
    </group>
</launch>
